source/js/package.json

```diff
@@ -32,11 +27,22 @@
     "constellation",
     "orbit"
   ],
-  "author": "Donald Cross",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/cosinekitty/astronomy/issues"
+  "files": [
+    "esm/astronomy.js",
+    "astronomy.d.ts",
```

Comment:
@matheo (I'm asking this because I don't know much about npm packages.) Should we also include astronomy.ts in this list? Do people ever develop against TypeScript source directly within an npm package?

Reply:
nope, ESM is already close to the ts syntax
and npmJS is not supposed to have TS packages :sweat_smile: